app/src/main/java/com/example/cocktails/ItemDetailsActivity.kt

```diff
@@ -16,7 +16,9 @@ class ItemDetailsActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.tabbed_item)
 
-        val sectionsPagerAdapter = SectionsPagerAdapter(this, supportFragmentManager)
+        val cocktail = intent.getParcelableExtra<Cocktail>("cocktail")
+
+        val sectionsPagerAdapter = SectionsPagerAdapter(this, supportFragmentManager, cocktail)
```

Comment:
@shahafhermann maybe change to: cocktail!! (not a nullable object)

Reply:
Yeah why not. Couldn't hurt.